app/models/participant_profile.rb

```diff
@@ -116,6 +114,13 @@ def sit_mentor?
   def policy_class
     ParticipantProfilePolicy
   end
+
+  def start_term_human
+    # Shouldn't we just update records in the database instead?
+    return "Autumn 2021" if start_term.blank?
```

Comment:
Can `start_term` be blank? It looks like we have a default set and `null: false`

Reply:
That is, I don't think you need to update the model because it's already set, and this method becomes unnecessary.

Reply:
Ah interesting - I wonder why we had all the `start_term || "Autumn 2021"` in the code. But - in any case, the default is incorrect - it should be `"autumn_2021"` to fit all the other values...

Reply:
I thought there was a bunch of records that didn't have any start_term date set? I might have that wrong but i thought we added the attribute and select a term date page after we had users. That's why I had the `start_term || "Autumn 2021"`

Reply:
@BroiSatse Although on second pass this is for a _human_ version of the `start_term` — e.g., not the form value. I _think_ the entire method here is not needed: can just depend on the default and use `humanize_start_term(start_term)` ?